src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/MarlinInterface.java

```diff
@@ -57,25 +60,19 @@
 	private Timer timeoutChecker = new Timer(10000,(e)->onTimeoutCheck());
 	private long lastReceivedTime;
 	
-	public MarlinInterface() {
+	public MarlinInterface(ConnectionButton connectionButton) {
```

Comment:
I'm not a fan of blowing up default constructors.  I'm not sure why the new PlotterControls are changing MarlinInterface, which wasn't broken and is only visible to advanced users...?

Reply:
`MarlinInterface` contained the button to connect to the robot. It is needed to link the serial transport to the interface. Now with this modification as the button is outside this interface, I have to pass it as an argument.